locust_tasks/locustfile.py

```diff
@@ -48,6 +48,27 @@ class Page(Enum):
                        '',
                        ''
                       )
+    SELECT_ADDRESS = ('<title>Select your address - Census 2021</title>',
+                      '<h1 class="question__title">Select your address</h1>',
+                      'I cannot find my address')
+    SELECT_METHOD = ('<title>How would you like to receive a new access code? - Census 2021</title>',
+                     '<h1 class="question__title">How would you like to receive a new household access code?</h1>',
+                     'To request a census in a different format or for further help, please')
+    ENTER_MOBILE = ('<title>What is your mobile phone number? - Census 2021</title>',
+                    '<h1 class="question__title">What is your mobile phone number?</h1>',
+                    'Continue')
+    CONFIRM_MOBILE = ('<title>Is this mobile phone number correct? - Census 2021</title>',
+                      '<h1 class="question__title">Is this mobile phone number correct?</h1>',
+                      'Continue')
+    ENTER_NAME = ('<title>What is your name? - Census 2021</title>',
+                  '<h1 class="question__title">What is your name?</h1>',
+                  'Continue')
+    CONFIRM_NAME = ('<title>Do you want to send a new access code to this address? - Census 2021</title>',
+                    '<h1 class="question__title">Do you want to send a new household access code to this address?</h1>',
+                    'Continue')
+    CODE_SENT = ('<title>We have sent an access code - Census 2021</title>',
```

Comment:
It ought to be possible to have an extraction start/end specified. If we fail when on this page then we probably only need to know where the code was sent to

Reply:
I did try but I don't think there was much choice for it. I'll have another look.

Reply:
The easiest way to do this is force the test to fail by expecting a number that won't be there. Then look at the failing output around the known phone number to decide on the fixed extraction start & end text.
If that doesn't work copy+paste the whole page text to a tmp html file and view in the browser.

Reply:
sounds like a good technique - I'll try it

Reply:
I re-discovered the reason why I hadn't put any start or end text in the CODE_SENT page; it was because the same page is used for both SMS and Post journeys but there are virtually no similarities in terms of what gets displayed underneath the title. However, after using your technique above, I noticed that they do both display their different text in the same style of box. So I've added the start and end tags for that box to the CODE_SENT page now. And I've passed in the hard coded phone number just for the time being (until the SampleGenerator card has been done).

Reply:
I've now changed it to use the phone number from the test data